dropwizard-metrics/3.1/src/main/java/com/wavefront/integrations/metrics/WavefrontReporter.java

```diff
@@ -185,6 +187,69 @@ public Builder withJvmMetrics() {
     }
 
     /**
+     * Builds a {@link WavefrontReporter} from the VCAP_SERVICES env variable, sending metrics
+     * using the given {@link WavefrontSender}. This should be used in PCF environment only.
+     *
+     * @param proxyServiceName The name of the wavefront proxy service. If wavefront-tile is used
+     *                        to deploy the proxy, then the service name will be 'wavefront-proxy'.
+     * @param failOnError A flag to determine what to do if the service parameters are not
+     *                    available. If 'true' then the method will throw RuntimeException else
+     *                    it will log an error message and continue.
+     * @return a {@link WavefrontReporter}
+     */
+    public WavefrontReporter bindToCloudFoundryService(String proxyServiceName,
```

Comment:
add an overloaded method that assumes proxyServiceName is "wavefront-proxy"? (perhaps also overloaded methods where failOfError is false too?)